webapp/README.md

```diff
@@ -10,7 +10,21 @@
 
 # Development
 
+## Pre-Requisites
```

Comment:
Please extract a commit for README change. This is unrelated to width change in history.tsx.